source/includes/important-delayed-replica-set-members.rst

```diff
@@ -0,0 +1,27 @@
+.. important::
+
+   If your replica set contains :doc:`delayed members
+   </core/replica-set-delayed-member>` ensure that the delayed
+   members are hidden and non-voting. 
+
+   Hiding delayed replica set members prevents applications from seeing 
+   and querying delayed data without a direct connection to that member.
+   Making delayed replica set members non-voting means they will not 
+   count towards acknowledging write operations with read concern 
+   :readconcern:`"majority"`.
+
+   If you do not hide delayed members and one or more nodes 
+   becomes unavailable, the replica set has to wait for the delayed 
+   member and the commit point lags. A lagged commit point can lead to
+   performance issues.
+
+   For example:
+
+   - Primary-Secondary-Delayed replica set config where delayed 
+     secondary is voting with 10 mins delay.
+   - With one non-delayed secondary unavailable, the degraded config of
+     Primary-Delayed will have to wait at least 10 minutes for a write 
+     concern with majority to be acknowledged.
+   - The majority commit point will take longer to advance leading to 
+     cache pressure similar to performance issues with a PSA replica 
+     set.
```

Comment:
Can we end this thought with something like:

> For more information on the majority commit point, see [Causal Consistency and Read and Write Concerns](https://docs.mongodb.com/manual/core/causal-consistency-read-write-concerns/)

I think "majority commit point" is a little confusing and providing that link for context may be useful.

I'm also not sure I understand the bit about a PSA replica set. We haven't mentioned them at all in this page and I feel like I'm missing some context. Is there anywhere we can link to in the docs that explains more about these performance issues? It kind of seems to come out of nowhere. We should also expand the acronym since users might not know what we mean by a PSA replica set.

Reply:
Updated / added a link to the replica set maintenance tutorial, which goes into more context of resolving the performance issues detailed with the commit point.